crates/forge/tests/it/config.rs

```diff
@@ -65,23 +65,27 @@ impl TestConfig {
             eyre::bail!("empty test result");
         }
         for (_, SuiteResult { test_results, .. }) in suite_result {
-            for (test_name, result) in test_results {
+            for (test_name, mut result) in test_results {
                 if self.should_fail && (result.status == TestStatus::Success) ||
                     !self.should_fail && (result.status == TestStatus::Failure)
                 {
                     let logs = decode_console_logs(&result.logs);
                     let outcome = if self.should_fail { "fail" } else { "pass" };
                     let call_trace_decoder = CallTraceDecoderBuilder::default().build();
```

Comment:
not related to this PR but I think we can decode traces using `self.known_contracts` here because currently this does nothing